xml-binding/3.0/_index.md

```diff
@@ -34,19 +34,20 @@ Please reference that ballot for the official results.
 
 ## Release Review
 
-The Specification Committee Ballot concluded successfully on 2020-mm-dd with the following results.
+The Specification Committee Ballot concluded successfully on 2020-10-26 with the following results.
 
 | Representative                                 | Representative for: | Vote |
 |------------------------------------------------|---------------------|------|
-| Kenji Kazumura, Michael DeNicola               | Fujitsu             |      |
-| Dan Bandera, Kevin Sutter                      | IBM                 |      |
-| Bill Shannon, Ed Bratt                         | Oracle              |      |
-| Mark Wareham, Steve Millidge                   | Payara              |      |
-| Scott Stark, Mark Little                       | Red Hat             |      |
-| David Blevins, Cesar Hernandez                 | Tomitribe           |      |
-| Ivar Grimstad                                  | EE4J PMC            |      |
-| Alex Theedom                                   | Participant Members |      |
-| Werner Keil                                    | Committer Members   |      |
-|                                                | Total               |      |
-
-The ballot was run in the [jakarta.ee-spec mailing list]()
+|Kenji Kazumura                                  | Fujitsu | +1
+|Dan Bandera, Kevin Sutter                       | IBM | +1
+|Ed Bratt, Dmitry Kornilov                       | Oracle | +1
+|Andrew Pielage, Matt Gill                       | Payara | +1
+|Scott Stark, Mark Little                        | Red Hat | +1
+|David Blevins, Jean-Louis Monteiro, Cesar Hernandez | Tomitribe | +1
```

Comment:
Nit, but the votes from Tomitribe are either from David or Jean-Louis.  Cesar is helping with mentoring, but not the voting.